src/BlockchainTestsFiller/bcUncleHeaderValiditiy/uncleBloomNot0_2Filler.json

```diff
@@ -88,6 +77,7 @@
 		"expectExceptionEIP150" : "InvalidStateRoot",
 		"expectExceptionEIP158" : "InvalidStateRoot",
 		"expectExceptionByzantium" : "InvalidStateRoot",
+		"expectExceptionConstantinople" : "InvalidStateRoot",
 		"blocknumber" : "3",
```

Comment:
same here. some tests are done with this block to pass on previous forks. (so there are no expectException) 
and other are just a copy but with this block passing on Byzantium / Constantinople. so stateRoot needs to be changed